secret.py

```diff
@@ -0,0 +1 @@
+API_KEY = '4b1c1bcde2mshd52c8eda876348dp1e7e33jsn6396e9edde23'
```

Comment:
Store this key in an environment variable. Delete the secret.py file from the application since we wont need it.
https://www.nylas.com/blog/making-use-of-environment-variables-in-python/